apps/site/components/EOL/EOLReleaseTable/index.tsx

```diff
@@ -1,10 +1,12 @@
+import Switch from '@node-core/ui-components/Common/Switch';
 import { getTranslations } from 'next-intl/server';
 import type { FC } from 'react';
 
 import provideReleaseData from '#site/next-data/providers/releaseData';
 import provideVulnerabilities from '#site/next-data/providers/vulnerabilities';
 import { EOL_VERSION_IDENTIFIER } from '#site/next.constants.mjs';
 
+import styles from './index.module.css';
```

Comment:
nit: style imports must happen after all other imports

Reply:
That's literally how all other components are doing, style imports on the bottom... If it is not erroring, it should.